### ISSUE TITLE
Webapp - contrast ratio link : ratio parameter is empty

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Asqatasun#119]
## Observed behaviour

![selection_037](https://cloud.githubusercontent.com/assets/6709977/18233577/70fb95c6-72eb-11e6-88ec-132474331cb4.png)

When you come on contrast finder from Asqatasun issue, the "ratio" parameter does not work.

I test a page on Asqatasun with RGAA 3. I see some contrast issues. When I clic on a contrast finder solution link, I'm correctly redirect to Contrast finder with the wrong colors but the ratio isn't 4.5.

In the URL, we can see that ratio parameter is empty...
`/result.html?foreground=%23FFFFFF&background=%23DB8000&ratio=&algo=HSV`
## Expected behaviour

Correct URL
`/result.html?foreground=%23FFFFFF&background=%23DB8000&ratio=4.5&algo=HSV`
## Solution

correct the line n° 159 of the `data-representation-5.jsp` file
[web-app/asqatasun-web-app/src/main/webapp/WEB-INF/view/data-representation/data-representation-5.jsp](https://github.com/Asqatasun/Asqatasun/blob/develop/web-app/asqatasun-web-app/src/main/webapp/WEB-INF/view/data-representation/data-representation-5.jsp#L159)

 in particular this code: 
`&ratio=${testResult.colorTestContrastRatio}`

**Comment by dzc34** [comment]
ping @koj @mfaure

**Comment by dzc34** [comment]
```
<td class="${evidenceElement2.key}">
<%-- (1) INITIAL CODE    ----> not works ----> @@@TODO fixme
    <c:set var="cfRatio" value="${testResult.colorTestContrastRatio}"/>
--%>
<%-- (1) Dirty alternative ----> it work ----> @@@TODO fixme  (START) --%>
    <c:choose>
        <c:when test="${testCode == 'Rgaa30-3-3-3'     ||
                        testCode == 'Rgaa30-3-3-4'     ||
                        testCode == 'Rgaa32016-3-3-3'  ||
                        testCode == 'Rgaa32016-3-3-4'     }" >
            <c:set var="cfRatio" value="3"/>
        </c:when>
        <c:when test="${testCode == 'Rgaa30-3-4-1'     ||
                        testCode == 'Rgaa30-3-4-2'     ||
                        testCode == 'Rgaa32016-3-4-1'  ||
                        testCode == 'Rgaa32016-3-4-2'     }" >
            <c:set var="cfRatio" value="7"/>
        </c:when>
        <c:otherwise>
            <c:set var="cfRatio" value="4.5"/>
        </c:otherwise>
    </c:choose>
<%-- (1) Dirty alternative  (END) --%>

    <c:set var="cfURL" value="${configProperties['contrastfinderServiceUrl']}/result.html"/>
    <a title="${cfLogoDescription}"
       href="${cfURL}?foreground=${fn:replace(fgColorHexa, "#", "%23")}&background=${fn:replace(bgColorHexa, "#", "%23")}&isBackgroundTested=false&ratio=${cfRatio}&algo=Rgb"
       rel="noreferrer noopener"
       target="_blank"><img alt="${cfLogoDescription}" style="margin-left: 15px;" src="<c:url value="/Images/contrast-finder.png"/>"/></a>
</td>
```

**Comment by dzc34** [comment]
fixed 79d228577f3c27b62ef6ce81df13b9ed99730159 (dirty fix)
